### DISCUSSION TITLE
Latest EP messes up taskbar windows

### DISCUSSION THREAD

**Discussion by mistertheloaf** [valinet/ExplorerPatcher#3814] [General]
Windows 11 22H2 - EP 22621.3880.66.6

Taskbar looks like this:

![ep](https://github.com/user-attachments/assets/fdfe1867-df5d-4c92-ade2-9a6c3cff302a)

That is good.  That is what I want.  

But when I install  EP 22621.3880.66.7  all those taskbar windows are reduced to just icons.  Not good.  Not what I want.  Is there a setting somewhere that I am missing?

**Comment by pyrates999** [comment]
Open up EP Properties application, go to taskbar.  Look at the options:

![image](https://github.com/user-attachments/assets/f2f54bc8-5266-4bb4-a684-4e15aa59697a)

**Reply by mistertheloaf** [reply]
> Open up EP Properties application, go to taskbar. Look at the options:
> 
> ![image](https://private-user-images.githubusercontent.com/106135298/377250528-f2f54bc8-5266-4bb4-a684-4e15aa59697a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjkxMjc0MTUsIm5iZiI6MTcyOTEyNzExNSwicGF0aCI6Ii8xMDYxMzUyOTgvMzc3MjUwNTI4LWYyZjU0YmM4LTUyNjYtNGJiNC1hNjg0LTRlMTVhYTU5Njk3YS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDE3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAxN1QwMTA1MTVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02NjEzMGI4OGFhMGJjZjVkMzEyNzliZDQwZWQ0Y2RlNGUzMDRmMzI2MTc4M2VjMTY5ODAyZWM5MjBmOGI3ZGJmJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.dnGCrxt2HywOpnqG_cqwh45aFyTLhaVYZK_rIy_yHKY)

Yes, the default for those options have been changed.

In previous versions of EP the default was "Never Combine" and now the default has been changed to "Always Combine".

**Reply by pyrates999** [reply]
Did you change them to what you want?

**Reply by mistertheloaf** [reply]
> Did you change them to what you want?

Yes.

I always had it set to "Never combine" which used to be the default.  But for some reason, the default in EP 22621.3880.66.7 is now "Always Combine".

It is an easy fix but you really shouldn't change default settings like that.

**Reply by pyrates999** [reply]
Then please close this.
